### DISCUSSION TITLE
Using crio cmdline, how can I check the current log_level set for crio?

### DISCUSSION THREAD

**Discussion by alpana17** [cri-o/cri-o#7845] [Q&A] [answered]
Using crio cmdline, how can I check the current log_level set for crio?

**Comment by kwilczynski** [comment, marked as answer]
Hi @alpana17, thank you for asking!

With any modern version of CRI-O, you can use the `crio` binary itself to interrogate current runtime configuration, for example:

- Check the _current_ log level set

**Note:** This requires access permissions to the CRI-O control Unix socket!

```console
# crio status config 2>/dev/null | grep log_level
    log_level = "info"
```

You can also use `c` for short of `config` in the above.

- Save the _current_ configuration to a file

```console
# crio status config > crio.conf
```

- Save annotated (with comments) _default_ configuration file

```console
$ crio config 2>/dev/null > crio.conf
```

Older versions of CRI-O ship with a dedicated command-line (CLI) utility called `crio-status`, which had the same option called `config` (or `c` for short), allowing the current runtime configuration to be printed.

**Reply by alpana17** [reply]
Thank you.
Got the log_level with cmd
crio-status config 2>/dev/null | grep log_level
